lib/modules.nix

```diff
@@ -1034,7 +1034,8 @@ let
   */
   mkRemovedOptionModule = optionName: replacementInstructions:
     { options, ... }:
-    { options = setAttrByPath optionName (mkOption {
+    { key = "lib/modules.nix#mkRemovedOptionModule-${showOption optionName}";
```

Comment:
```suggestion
    { _file = "lib/modules.nix"; key = "mkRemovedOptionModule-${showOption optionName}";
```

Probably?

Reply:
It seemed to me that `key` defaults to `_file` normally, so I assumed that it's not possible to pass `_file` and `key` to `disabledModules` to get more precise filtering, but I'd need to confirm that suspicion.

Reply:
I don't think that's possible, but I think `mkRemovedOptionModule-` is precise enough, and `_file` might help with error messages (?)

Reply:
`_file` is for error messages.

By setting `_file` we would hide the importing module's `_file`, which is the `_file` we actually want in error messages.
Usually we have
```nix
# foo.nix
{
  imports = [ (mkRemovedOptionModule [ "bar" ]) ];
}
```

and then `error in foo.nix` is more useful than `error in lib/modules.nix` or `error in removed option bar`.

So we should only set `key`.